src/tasks/daily/daily_trade_mixin.py

```
@@ -453,12 +460,15 @@ def buy_sell(self, target_areas=None, keep_area_context=False, after_buy=None):
                     continue
                 self.wait_ui_stable(refresh_interval=1)
                 if self.plus_max():
-                    self.wait_click_ocr(
+                    # 仅当出售确认按钮真正点击成功且弹窗关闭，才算真正卖出
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)